cfme/common/vm_console.py

```diff
@@ -237,7 +237,9 @@ def switch_to_appliance(self):
     def switch_to_console(self):
         """Switch focus to console tab/window."""
         logger.info("Switching to console: window handle = {}".format(self.console_handle))
-        self.browser.selenium.switch_to.window(self.console_handle)
+        if (self.console_handle is not None and self.console_handle in
```

Comment:
In case of VMRC Console, the browser window for console is short-lived, hence it becomes important to check if the `console_handle` exists in available `selenium.window_handles` before attempting to switch to console or close the windw, hence this check.